src/components/Common/CustomButton/CustomButton.jsx

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import classes from './CustomButton.module.css';
+const CustomButton = (props) => {
+    return (
+        <div>
+            <button
+                style={{
+                    color: props.styles.color,
+                    backgroundColor: props.styles.backgroundColor,
+                    height: props.styles.height || "32px",
+                    width: props.styles.width || "100",
+                    border: `1px solid ${props.styles.border}` || "",
```

Comment:
let's give the user to add the entire border. User gives "4px dashed green" directly as a prop.
so change -> 
border: `1px solid ${props.styles.border}` || ""
to
border : props.styles.border